apps/api/src/lib/directus-adapter.ts

```diff
@@ -493,6 +493,9 @@ export function toPackBase(
 
   return {
     allowBidExpiration: template.allow_bid_expiration,
+    additionalImages: isStringArray(template.additional_images)
+      ? template.additional_images.map((id) => getFileURL(id))
+      : template.additional_images.map((t) => getFileURL(t.directus_files_id)),
```

Comment:
Small thing, what does `t` stand for? Maybe use `ai` for additional_images or `df` for directus file?